code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -236,7 +236,7 @@
 
 /obj/item/weapon/gun/ballistic/automatic/carbine
 	name = "\improper Marine Carbine"
-	desc = "Standard issue weapon used by United Earth Government Marines in space to space combat. Due to modifications for space combat, the weapon has a very limited firing range."
+	desc = "Standard issue weapon used by UEG Marines in space to space combat. Due to modifications for space combat, the weapon has a very limited firing range."
```

Comment:
Intentional, you have to go our of your way to read it and its not the full naming. I think its a appropriate desc since it explains the weapons existence and why it has limited range.